packages/editor-ui/src/components/CredentialsInput.vue

```diff
@@ -178,8 +190,33 @@ export default mixins(
 			nodesAccess: [] as string[],
 			name: '',
 			propertyValue: {} as ICredentialDataDecryptedObject,
+			credentialsParams: {
+				nodeType: `n8n-nodes-base.${this.credentialTypeData.documentationUrl}`, // workaround
+				credentialsName: this.credentialTypeData.name,
+			},
 		};
 	},
+
+	/**
+	 * Before mounting, load the translations for this node (if any),
+	 * which may contain credentials parameters.
+	 *
+	 * This is necessary in case the node has not been placed on the canvas,
+	 * so its translations are not yet available.
+	 */
+	beforeMount() {
+		return new Promise<void>(resolve => {
+			this.restApi().getNodesInformation([this.credentialsParams.nodeType])
+				.then(nodeInfo => {
+					if (nodeInfo[0] && nodeInfo[0].translation) {
+						const nodeTranslations = nodeInfo[0].translation;
+						if (nodeTranslations) addNodeTranslations(nodeTranslations);
+						resolve();
+					}
+				});
+		});
+	},
```

Comment:
Only the zeroth item because the call returns an array and the node info to be fetched is for the single node whose credential is being edited, i.e. `this.credentialsParams.nodeType`.

The suggestion cannot be implemented, seemingly because `credentialsParams` does not exist on `this` of type `CombinedVueInstance` at that point in the lifecycle. The `Promise` is a workaround to be able to refer to it.